features/Steps/ResponseSteps.php

```diff
@@ -308,4 +321,42 @@ private function removeDates(string $value): string
         $datePattern = '/(?<=dcterms:created\s|dcterms:modified\s)"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{2}:\d{2}"/';
         return preg_replace($datePattern, '""', $value);
     }
+
+    /**
+     * @Then I calculate the image hash with description :description, copyright :copyrightHolder and language :language for :imageId
+     */
+    public function iCalculateTheImageHashWith(string $description, string $copyrightHolder, string $language, string $imageId): void
+    {
+        switch ($imageId) {
+            case 'imageId':
```

Comment:
Why is this weird switch/case added? What will happen if there are 3 images?

Reply:
It is there because some tests require 1 image, some required 2 images.
I am open for better suggestions.

Reply:
Maybe that is no longer needed when the `@Transform` method is used? Or is the issue that you can't replace the correct hash for the correct image?

Reply:
Maybe you need to add the name of the image hash placeholder that needs to be replaced?